### ISSUE TITLE
issue starting the SSE. 

### ISSUE THREAD

**Issue by tanvij** [nabeel-oz/qlik-py-tools#126]
**Describe the bug**
A clear and concise description of the bug.

**Screenshots**
If applicable, add screenshots from `Qlik-Py-Init` to show the full error message.

**Environment:**
 - OS [e.g. Windows Server, Linux]
 - Qlik Sense Enterprise/ Desktop/ Kubernetes/ QlikView
 - SSE Version

**Additional context**

**Comment by tanvij** [comment]
Qlik-Py-Init ran as below screenshot  
![image](https://user-images.githubusercontent.com/13887324/170001540-5fdecd63-1136-4f54-b5b7-5dd9736367cc.png)

ignoring the error with few libraries i went ahead to run the Qlik_Py_Start file. It gave error: no module named google as below screenshot
![image](https://user-images.githubusercontent.com/13887324/170001851-3c270518-2a65-4914-9c67-97cd98c6c73e.png)

**Comment by tanvij** [comment]
Also tried activating virtual env in power shell:
C:\Users\002AB6744\Documents\qlik-py-tools-8.1\qlik-py-tools-8.1\qlik-py-env\pretrained> ..\Scripts\activate.ps1

Got below error:
..\Scripts\activate.ps1 : File
C:\Users\002AB6744\Documents\qlik-py-tools-8.1\qlik-py-tools-8.1\qlik-py-env\Scripts\Activate.ps1
cannot be loaded because running scripts is disabled on this system. For more information, see
about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ ..\Scripts\activate.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess

**Comment by abx-212** [comment]
Hello, have you found a solution to your error? If so, may you kindly share it
I'm facing the same problem with installation

**Comment by tanvij** [comment]
No, i didnt find the solution and finally dropped to use this SSE as I
found that libraries used are obsolete.

Thanks,
Tanvi Jain

**Comment by abx-212** [comment]
Thank you for sharing,
Did you find an alternative solution, or another SSE?

**Comment by tanvij** [comment]
No ...I could not use python and machine learning models within qlik

**Comment by MarkJM84** [comment]
Hi, we had the same problem a while ago:
https://github.com/nabeel-oz/qlik-py-tools/issues/122

We came to the weird conclusion that it might be a hardware issue.
I can't find the documentation we had, but there are older server processors that don't meet the requirements for the python tensorflow module.

Might this can point you to the right direction to find a solution.

**Comment by abx-212** [comment]
> 

Thanks a lot, Mark! the issue you've raised was helpful.

**Comment by rowenagold** [comment]
I have this same issue after following the steps very clearly. 
Furthermore ive tried other steps with the following error

 pip install google-protobuf
ERROR: Could not find a version that satisfies the requirement google-protobuf (from versions: none)
ERROR: No matching distribution found for google-protobuf

 pip show protobuf
Name: protobuf
Version: 4.24.3
Summary:
Home-page: https://developers.google.com/protocol-buffers/
Author: protobuf@googlegroups.com
Author-email: protobuf@googlegroups.com
License: 3-Clause BSD License

i have protobof and its installed in the same path as all other python library installed. 

Is there some other option for SSE on Qlik with python?
